src/controllers/SoundController.ts

```diff
@@ -0,0 +1,43 @@
+const sound0 = require('../../static/sounds/0.mp3');
+const sound1 = require('../../static/sounds/1.mp3');
+const sound2 = require('../../static/sounds/2.mp3');
+const soundCapture = require('../../static/sounds/capture.mp3');
+
+import { SoundTypes } from '../enums/SoundTypes';
+
+class SoundController {
+    private lastPlayedMoveSound: number = 0;
+
+    public makeSound(soundType: SoundTypes) {
+        switch (soundType) {
+            case SoundTypes.NORMAL_MOVE:
+                this.playNormalMoveSound();
+                break;
+            case SoundTypes.CAPTURING_MOVE:
+                this.playCapturingMoveSound();
+                break;
+            default:
+                break;
+        }
+    }
+
+    private playCapturingMoveSound(): void {
+        let sound = new Audio('./static/sounds/capture.mp3');
+        sound.play();
+    }
+
+    private playNormalMoveSound(): void {
+        const generateRandomNumber = () => Math.floor(Math.random() * 3);
```

Comment:
Ta trójka troszkę magiczna. Domyślam się że jest tutaj bo mamy trzy różne dźwięki na okoliczność poruszenia pionka jeżeli kiedyś uznamy że chcemy dodać nowy dźwięk lub usunąć istniejący będziemy musieli pamiętać by wejść do tej metody i to zmienić. Dlatego proponuje by wyciągnąć sobie to jako stałą tej klasy.